modules/tinymce/src/plugins/nonbreaking/test/ts/webdriver/NonbreakingTypingTest.ts

```diff
@@ -16,6 +16,8 @@ describe('webdriver.tinymce.plugins.nonbreaking.NonbreakingTypingTest', () => {
     base_url: '/project/tinymce/js/tinymce'
   }, [ Plugin ]);
 
+  const tester = Env.browser.isSafari() ? it.skip : it;
```

Comment:
Can we put comment and potentially a link to a JIra that can clarify why this has to be done?